src/primary/primary_client.cpp

```diff
@@ -31,6 +31,7 @@
 #include <ur_client_library/primary/primary_client.h>
 #include <ur_client_library/primary/robot_message.h>
 #include <ur_client_library/primary/robot_state.h>
+#include "ur_client_library/exceptions.h"
 
```

Comment:
```suggestion
#include <ur_client_library/helpers.h>
```
In my opinion we should be explicit and inlcude the helpers.h file here for the wait_for function. I am however a bit surprised that it doesn't complain when compiling.

Reply:
It probably gets pulled in recursively. But I agree.